hw4/hw4-v1tal4ik/src/v1tal4ik.js

```diff
@@ -0,0 +1,94 @@
+// 1. Turn given object into iterable so it was possible to loop through it via for...of construction.
+const range = {
+  from: 1,
+  to: 10,
+};
+
+range[Symbol.iterator] = function () {
+  let currentValue = this.from;
+  const lastValue = this.to;
+
+  return {
+    next() {
+      if (currentValue <= lastValue) {
+        return {
+          done: false,
+          value: currentValue++,
+        };
+      }
+      return { done: true };
+    },
+  };
+};
+
+for (const value of range) {
+  console.log(value);
+}
+
+
+// 2. Loop through iterable object created in task 1 and create an array which contains only odd numbers. Expected result:
+// 3. Create array which contains only even numbers from iterable created in task 1 with some other method than in task 2. Expected result:
+const arrOfOddNumbers = [];
+const arrOfEvenNumbers = [];
+
+for (const value of range) {
+  (value % 2 === 0) ? arrOfEvenNumbers.push(value) : arrOfOddNumbers.push(value);
```

Comment:
actually in third task i asked to perform this operation with some other way than the second one. So please figure out something else for that 😉